BraveRewardsUI/Settings/Auto-Contribute/Details/AutoContributeDetailsViewController.swift

```diff
@@ -174,23 +174,15 @@ extension AutoContributeDetailViewController: UITableViewDataSource, UITableView
         // Monthly payment
         guard let wallet = state.ledger.walletInfo else { break }
         let monthlyPayment = state.ledger.contributionAmount
-        let choices = wallet.parametersChoices.map { $0.doubleValue }
-        let selectedIndex = choices.firstIndex(of: monthlyPayment) ?? 0
-        let stringChoices = choices.map { choice -> String in
-          var amount = "\(choice) \(Strings.BAT)"
-          if let dollarRate = state.ledger.dollarStringForBATAmount(choice) {
-            amount.append(" (\(dollarRate))")
+        let choices = wallet.parametersChoices.map { BATValue($0.doubleValue) }
+        let selectedIndex = choices.map({ $0.doubleValue }).firstIndex(of: monthlyPayment) ?? 0
+        
+        let controller = BATValueOptionsSelectionViewController(ledger: state.ledger, options: choices, isSelectionPrecise: false, selectedOptionIndex: selectedIndex) { [weak self] (selectedIndex) in
```

Comment:
can we split this line, it's pretty long, also brackets around `selectedIndex` can be removed